style.css

```diff
@@ -0,0 +1,334 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+}
+
+html {
+  font-size: 15px;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a:hover,
+.fa {
+  cursor: pointer;
+}
+
+section.header {
+  background-color: white;
+  border-bottom-left-radius: 7.33rem;
+  padding-top: 4.6rem;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 1.7rem;
+  padding: 0 1.6rem;
+}
+
+.logo {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.2rem;
+}
+
+.logo a {
+  color: #6070ff;
+}
+
+.nav-links {
+  display: none;
+}
+
+.fa-bars {
+  color: #6070ff;
+}
+
+div.headline {
+  padding: 7.6rem 1.6rem;
+  background-image: url("/images/header-shapes-mobile.png");
+  background-size: cover;
+  background-position: left;
+  display: flex;
+  flex-direction: column;
+  gap: 0.8rem;
+}
```

Comment:
## Styling ✨✨

- [ ] **Kudos on using the ```background-image`` 👏 👏** However, I think it would be better to replicate the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=wMKvp4oyMqZ6YudA-0) as much as possible by using the same background size and position 🙏 
- [ ] It would be better also to replicate the margin and padding given by the Figma design, 🙏 
- [ ] It would be better also to increase the size of your Icons to replicate the Given design as much as possible 

**_I have highlighted some areas to be improved for your reference 👇_** 

  **Your design** | **Figma design**
   | --- | --- |
   ![image](https://user-images.githubusercontent.com/102757126/227290416-db10acc7-b4f9-4f87-99f0-0019a85357a2.png) | ![image](https://user-images.githubusercontent.com/102757126/227283664-c5985c5f-cede-4565-b5c8-c5c87ef73573.png)